packages/@react-spectrum/provider/src/Provider.tsx

```diff
@@ -69,7 +69,12 @@ function Provider(props: ProviderProps, ref: DOMRef<HTMLDivElement>) {
     isRequired,
     isReadOnly,
     validationState
-  });
+  };
+  let filteredProps = {};
+  Object.entries(currentProps).forEach((aProp) => aProp[1] !== undefined && (filteredProps[aProp[0]] = aProp[1]));
```

Comment:
we can make it easier to read
`Object.entries(currentProps).forEach(([key, value]) => value !== undefined && (filteredProps[key] = value));`